README.md

```diff
@@ -350,14 +353,92 @@ export class AppComponent {
 }
 ```
 
+#### <a id="selectItems"></a> `selectItems(predicate)`
+
+Iterates over collection of `SelectItemDirective`'s, selecting all items from that collection that meet the condition specified in the `predicate` function. The `predicate` is invoked with one argument `item`, which is equivalent to the value returned by `SelectItemDirective.value`.
+
+**Arguments**
+
+`predicate (Function)`: The function invoked per iteration.
```

Comment:
What about `predicate ((item: T) => boolean)`,  instead of `predicate (Function)`